doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgresql_sink.md

```diff
@@ -119,6 +120,9 @@ Regarding the specific data stored within the above table, if `attr_persistence`
 *  For each notified attribute, a field named as the attribute is considered. This field will store the attribute values along the time.
 *  For each notified attribute, a field named as the concatenation of the attribute name and `_md` is considered. This field will store the attribute's metadata values along the time.
 
+#### <a name="section1.2.6"></a>Native attribute type
+Regarding the specific data stored within the above table, if `attr_native_types` parameter is set to `true` then attribute is inserted using its native type, if `false` then will be strinfy.
```

Comment:
I'd suggest to add the new parameter also to the table:

![imagen](https://user-images.githubusercontent.com/1534240/59492837-17f3de80-8e8a-11e9-8e67-e7bcf92b9994.png)

Reply:
Where is that table?

Reply:
I think is here: https://github.com/telefonicaid/fiware-cygnus/blob/5e0da0489023e2b866d1d661d2d083bf5f7c07ab/doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgresql_sink.md#configuration

Reply:
fixed in 2cdb9f1b32c4567433b800c011250571abf4539b